components/sandbox/index.js

```diff
@@ -1,32 +1,51 @@
 /**
  * WordPress dependencies
  */
-import { Component, renderToString } from '@wordpress/element';
+import { Component, renderToString, createRef } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import FocusableIframe from '../focusable-iframe';
 
 export default class Sandbox extends Component {
 	constructor() {
 		super( ...arguments );
 
 		this.trySandbox = this.trySandbox.bind( this );
 		this.checkMessageForResize = this.checkMessageForResize.bind( this );
-		this.checkFocus = this.checkFocus.bind( this );
+
+		this.iframe = createRef();
 
 		this.state = {
 			width: 0,
 			height: 0,
 		};
 	}
 
+	componentDidMount() {
+		window.addEventListener( 'message', this.checkMessageForResize, false );
```

Comment:
withGlobalEvent?

Reply:
>withGlobalEvent?

Hah! But in seriousness, yes, it should! Will add.